src/rules/no-nodejs-modules.js

```diff
@@ -7,21 +9,53 @@ function reportIfMissing(context, node, allowed, name) {
   }
 }
 
+function isIgnored(ignored, filename) {
+  return ignored.some(c => (
+    minimatch(filename, c) ||
+    minimatch(filename, path.join(process.cwd(), c))
+  ))
+}
+
 module.exports = {
   meta: {
     docs: {},
+    schema: [{
+      type: 'object',
+      properties: {
```

Comment:
Awesome! Could we also set "allow additional properties" or whatever it's called to false?